src/components/views/dialogs/DeviceVerifyDialog.js

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2016 OpenMarket Ltd
```

Comment:
there is, because it's a bunch of code which was written for OM. The fact it is moving to a different file doesn't change that.